miden-lib/asm/miden/kernels/tx/tx.masm

```diff
@@ -8,9 +8,9 @@ use.miden::kernels::tx::note
 # =================================================================================================
 
 # Constants for different note types
-const.OFFCHAIN_NOTE=0
-const.ENCRYPTED_NOTE=1
-const.PUBLIC_NOTE=2
+const.PUBLIC_NOTE=1     # 0b01
+const.OFFCHAIN_NOTE=2   # 0b10
+const.ENCRYPTED_NOTE=3  # 0b11
```

Comment:
The values are now the bitmasks used to test the note's tag. The bits in the mask must be set to `0` in the note's tag.